ca/config.go

```diff
@@ -165,6 +173,13 @@ func (s *SecurityConfig) IssuerInfo() *IssuerInfo {
 	return s.issuerInfo
 }
 
+// SetWatch sets a queue on which changes to security config (roots and certificate issuer info) are published
+func (s *SecurityConfig) SetWatch(q *watch.Queue) {
```

Comment:
It seemed easier to set watch on the security config, so the setter can clean it up, as opposed to adding a cleanup function to the security config so the watch can be closed when it's done.  If we prefer that I can add it instead, but it means a lot more changes to the tests, etc.